tensorboard/plugins/text_v2/text_v2_plugin_test.py

```diff
@@ -0,0 +1,184 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
This is a forked file so maybe you it should have `2020`?

Reply:
done